Yoast/javascript
# Rename feature-toggle to feature-flag

## Summary

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo-premium/pull/2351

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Feature flags.

## UI changes

## Quality assurance


Fixes #244
